unstructured/chunking/title.py

```diff
@@ -56,8 +56,8 @@ def chunk_table_element(element: Table, max_characters: int = 500) -> List[Table
 def chunk_by_title(
     elements: List[Element],
     multipage_sections: bool = True,
-    combine_text_under_n_chars: int = 500,
-    new_after_n_chars: int = 500,
+    combine_text_under_n_chars: int | None = None,
```

Comment:
As far as I can tell, this syntax wasn't introduced until Python 3.10. On the other hand, CI isn't complaining (maybe because typing isn't really enforced?). Everywhere else we're using the `Optional[int]` syntax, I'm inclined to stick with that here.

Reply:
This syntax is accepted in Python 3.7+ (and maybe further back) when `from __future__ import annotations` appears in the import section.

This "pipe == Union" syntax is recommended as Union will be deprecated, so it's the "way forward" so to speak, which is why I started using it. Let me see if I can find a reference for that ... here's one: https://mail.python.org/archives/list/typing-sig@python.org/message/624BEVFNZXFEQP6ZWIGB4YKSPMCBE2M6/

next response in that thread is Guido approving.

They `from __future__ import annotations` also gives some other very nice benefits like not requiring forward references to be quoted and a few other things.

Reply:
Ok, fix is in for this @Alan :)

Reply:
Ok, fix is in for this @qued :)